src/osx/core/uilocale.mm

```diff
@@ -35,6 +35,14 @@
 extern wxString
 wxGetInfoFromCFLocale(CFLocaleRef cfloc, wxLocaleInfo index, wxLocaleCategory cat);
 
+// Helper function to retrieve a pointer to the NSLocale of the current wxUILocale
+static NSLocale* gs_nsloc = nullptr;
```

Comment:
This is really trivial, but I'd call it `gs_currentNSLocale` as extra clarity is worth the extra length for global variables IMHO.

Reply:
> This is really trivial, but I'd call it `gs_currentNSLocale` as extra clarity is worth the extra length for global variables IMHO.

Ok, I will apply this change.